src/test/java/com/zufar/onlinestore/cart/stub/CartDtoTestStub.java

```diff
@@ -0,0 +1,224 @@
+        shoppingSession.setProductsQuantity(6);
+        shoppingSession.setCreatedAt(OffsetDateTime.now());
+        shoppingSession.setClosedAt(OffsetDateTime.now().plusHours(2));
+
+        return shoppingSession;
+    }
+
+    public static ShoppingSessionDto createShoppingSessionDto () {
+        UUID userId = UUID.fromString("ebd4d43f-3152-4af5-86dd-526a002cbbc3");
+        ShoppingSessionDto shoppingSessionDto = new ShoppingSessionDto();
+        List<ShoppingSessionItemDto> items = new ArrayList<>();
+
+        UUID firstProductDtoId = UUID.fromString("a834c24e-886d-470f-bf19-7454a60f0639");
+        ProductInfoDto firstProductInfoDto = new ProductInfoDto();
+        firstProductInfoDto.setId(firstProductDtoId);
+        firstProductInfoDto.setName("First test name");
+        firstProductInfoDto.setDescription("First test description");
+        firstProductInfoDto.setPrice(BigDecimal.valueOf(1.1));
+        firstProductInfoDto.setQuantity(1);
+        firstProductInfoDto.setActive(true);
+        UUID secondProductId = UUID.fromString("2ade78e3-aa45-4b6b-adf4-86f8302ced7d");
+        ProductInfoDto secondProductInfoDto = new ProductInfoDto();
+        secondProductInfoDto.setId(secondProductId);
+        secondProductInfoDto.setName("Second test name");
+        secondProductInfoDto.setDescription("Second test description");
+        secondProductInfoDto.setPrice(BigDecimal.valueOf(2.2));
+        secondProductInfoDto.setQuantity(2);
+        secondProductInfoDto.setActive(true);
+
+        UUID firstItemId = UUID.fromString("9b588163-b781-46bf-8714-bd0145337ddc");
+        ShoppingSessionItemDto firstItemDto = new ShoppingSessionItemDto();
+        firstItemDto.setId(firstItemId);
+        firstItemDto.setProductInfo(firstProductInfoDto);
+        firstItemDto.setProductQuantity(1);
+        UUID secondItemId = UUID.fromString("e5cadeb1-089c-430f-85d1-e18438167241");
+        ShoppingSessionItemDto secondItemDto = new ShoppingSessionItemDto();
+        secondItemDto.setId(secondItemId);
+        secondItemDto.setProductInfo(secondProductInfoDto);
+        secondItemDto.setProductQuantity(2);
+
+        items.add(firstItemDto);
+        items.add(secondItemDto);
+
+        shoppingSessionDto.setUserId(userId);
+        shoppingSessionDto.setItems(items);
+        shoppingSessionDto.setItemsQuantity(2);
+        shoppingSessionDto.setProductsQuantity(3);
+        shoppingSessionDto.setItemsTotalPrice(BigDecimal.valueOf(5.5));
+        shoppingSessionDto.setCreatedAt(OffsetDateTime.now());
+        shoppingSessionDto.setClosedAt(OffsetDateTime.now().plusHours(2));
+
+        return shoppingSessionDto;
+    }
+
+    public static ShoppingSessionDto createFullShoppingSessionDto () {
+        UUID userId = UUID.fromString("ebd4d43f-3152-4af5-86dd-526a002cbbc3");
+        ShoppingSessionDto shoppingSessionDto = new ShoppingSessionDto();
+        List<ShoppingSessionItemDto> items = new ArrayList<>();
```

Comment:
Create a separate method which returns `List<ShoppingSessionItemDto> items`
And then use this method in tests

Reply:
same

Reply:
done